conduit/pschema/v1/fromproto/schema.go

```diff
@@ -27,23 +27,33 @@ func _() {
 	_ = cTypes[int(cschema.TypeAvro)-int(conduitv1.Schema_TYPE_AVRO)]
 
 	// Compatibility between the protocol's schema.Type and the Protobuf schema type
-	_ = cTypes[int(schema.TypeAvro)-int(conduitv1.Schema_TYPE_AVRO)]
+	_ = cTypes[int(pschema.TypeAvro)-int(conduitv1.Schema_TYPE_AVRO)]
 }
 
-func CreateRequest(req *conduitv1.CreateSchemaRequest) schema.CreateRequest {
-	return schema.CreateRequest{
-		Name:  req.Name,
-		Type:  schema.Type(req.Type),
-		Bytes: req.Bytes,
+func CreateRequest(req *conduitv1.CreateSchemaRequest) pschema.CreateRequest {
+	return pschema.CreateRequest{
+		Subject: req.Subject,
+		Type:    pschema.Type(req.Type),
+		Bytes:   req.Bytes,
 	}
 }
 
-func CreateResponse(resp *conduitv1.CreateSchemaResponse) schema.CreateResponse {
-	return schema.CreateResponse{
+func CreateResponse(resp *conduitv1.CreateSchemaResponse) pschema.CreateResponse {
+	return pschema.CreateResponse{
 		Instance: cschema.Instance{
-			ID:      resp.Schema.Id,
-			Name:    resp.Schema.Name,
-			Version: resp.Schema.Version,
+			Subject: resp.Schema.Subject,
+			Version: int(resp.Schema.Version),
+			Type:    cschema.Type(resp.Schema.Type),
+			Bytes:   resp.Schema.Bytes,
+		},
+	}
+}
+
+func GetResponse(resp *conduitv1.GetSchemaResponse) pschema.GetResponse {
+	return pschema.GetResponse{
+		Instance: cschema.Instance{
+			Subject: resp.Schema.Subject,
+			Version: int(resp.Schema.Version),
 			Type:    cschema.Type(resp.Schema.Type),
 			Bytes:   resp.Schema.Bytes,
 		},
```

Comment:
The conversion from a proto schema to a Go schema (and back) should be moved to conduit-commons, so we have methods like `FromProto` and `ToProto` on `schema.Schema` (see how we do it on [`opencdc.Record`](https://github.com/ConduitIO/conduit-commons/blob/main/opencdc/proto.go)). This way the code can be reused in the processor SDK as well.

Reply:
@lovromazgon https://github.com/ConduitIO/conduit-commons/pull/72